src/components/GridContainer/GridContainer.tsx

```diff
@@ -0,0 +1,129 @@
+import styled from "styled-components";
+import { HTMLAttributes } from "react";
+
+export type DisplayOptions = "grid" | "inline-grid";
+export type FlowOptions = "row" | "column" | "row-dense" | "column-dense";
+type GapOptions = "none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl" | "xxl" | "unset";
+type ItemsOptions = "start" | "center" | "end" | "stretch";
+type ContentOptions =
+  | "center"
+  | "space-between"
+  | "space-around"
+  | "space-evenly"
+  | "start"
+  | "stretch"
+  | "end"
+  | "left"
+  | "right";
+
+export interface GridContainerProps extends HTMLAttributes<HTMLDivElement> {
+  alignItems?: ItemsOptions;
+  alignContent?: ContentOptions;
+  children?: React.ReactNode;
+  display?: DisplayOptions;
```

Comment:
I think instead of display we can introduce inline to check if it is inline-grid or not
Adding display to grid is confusing